docs/proposals/config_data.md

```diff
@@ -0,0 +1,316 @@
+# Generic Configuration Object
+
+## Abstract
+
+This proposal proposes a generic configuration object named `ConfigData` stored inside `etcd` that
```

Comment:
AFAIK there are no items for this in 1.2, it should definitely be discussed in another issue. If you have a good idea of the specifications please create an issue, otherwise after we finalize this proposal, I'll try to gather the discussions about templating and create the issue.